src/reference-manual/types.Rmd

```diff
@@ -458,8 +458,7 @@ scalar, the existence of which is controlled by `include_alpha`.
 
 This coding pattern allows a single Stan program to define different
 models based on the data provided as input.  This strategy is used
-extensively in the implementation of the RStanArm package (see
-\url{http://mc-stan.org/users/interfaces/rstanarm}).
+extensively in the implementation of the RStanArm package (see (https://mc-stan.org/rstanarm)).
```

Comment:
Does this do the right thing with just parens?  It lists the URL and provides a link?  Otherwise, we want "[url](url)".

Reply:
good catch - linking directly to words 'RStanArm package'.